### PR TITLE
C356788 A user cannot stop running jobs with "Data import: Can view only" permission (folijet) (TaaS)

### PR DESCRIPTION
**Pull request by KhusniddinEpam**
[Test Rail](https://foliotest.testrail.io//index.php?/cases/view/356788)
[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5735/allure)
image

### CODE REVIEW

File: cypress/support/fragments/data_import/dataImport.js [hunk 1]
```diff
@@ -431,6 +431,16 @@ export default {
       });
   },
 
+  verifyTrashIconInvisibleForUser: (user) => {
+    cy.login(user.username, user.password, {
```
Comment (dnovikau) [inline cypress/support/fragments/data_import/dataImport.js:435]: Login and navigation actions does not related to DataImport module. I'd recommend to move these steps to test and leave only verification of button absense here.

File: cypress/e2e/data-import/permissions/cannot-stop-running-jobs-with-can-only-view-permission.cy.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    });
+
+    after('delete test data', () => {
+      cy.getAdminToken();
+      Users.deleteViaApi(userA.userId);
+      Users.deleteViaApi(userB.userId);
+    });
+
+    it(
+      'C356788 A user cannot stop running jobs with "Data import: Can view only" permission (folijet)',
```
Comment (dnovikau) [inline cypress/e2e/data-import/permissions/cannot-stop-running-jobs-with-can-only-view-permission.cy.js:42]: description should have '(TaaS)' at the end:
 C356788 A user cannot stop running jobs with "Data import: Can view only" permission (folijet) (TaaS)

File: cypress/e2e/data-import/permissions/cannot-stop-running-jobs-with-can-only-view-permission.cy.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      { tags: [TestTypes.extendedPath, DevTeams.folijet] },
```
Comment (dnovikau) [inline cypress/e2e/data-import/permissions/cannot-stop-running-jobs-with-can-only-view-permission.cy.js:43]: By the new agreement it should be replaced with array of strings:
```JS 
{ tags: ['extendedPath', 'folijet'] },
```

**Review by dnovikau — CHANGES_REQUESTED** [review]

File: cypress/support/fragments/data_import/dataImport.js (hunk 1 above)
Comment (KhusniddinEpam) [reply cypress/support/fragments/data_import/dataImport.js]: updated

File: cypress/e2e/data-import/permissions/cannot-stop-running-jobs-with-can-only-view-permission.cy.js (hunk 2 above)
Comment (KhusniddinEpam) [reply cypress/e2e/data-import/permissions/cannot-stop-running-jobs-with-can-only-view-permission.cy.js]: updated

File: cypress/e2e/data-import/permissions/cannot-stop-running-jobs-with-can-only-view-permission.cy.js (hunk 3 above)
Comment (KhusniddinEpam) [reply cypress/e2e/data-import/permissions/cannot-stop-running-jobs-with-can-only-view-permission.cy.js]: updated

**Review by dnovikau — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**